File: 자바 기초/source/day04/src/exam04/Calculator.java
package exam04;

public interface Calculator {
    int num = 10; // public static final - 정적 상수
    // public abstract   int add(int num1, int num2);
    // public abstract - 컴파일러가 자동 추가
    // 목적이 명확하게 정해져 있음
    // 다른 접근자는 사용 불가
    int add (int num1, int num2);
}

File: 리액트/hello/src/libs/calculator.js
function add(num1, num2) {
    return num1 + num2;
}

export function sub(num1, num2) {
    return num1 - num2;
}

 const PI = 3.141592;

 export{ sub, PI }

 

export default add; // add 함수 내보내기

// export default 변수명; -> 모듈 1개 내보낼때 사용